src/routes/fill-event/createFillEvent.ts

```diff
@@ -0,0 +1,49 @@
+import { FastifyInstance, FastifyReply, FastifyRequest } from 'fastify';
+import {
+  createFillEventBody,
+  CreateFillEventBody,
+  fillEventResponse,
+} from '../../types/fillEvent.types';
+import { errorHandler } from '../../lib/errorHandler';
+import { createFillEvent, calcTotalCost } from '../../lib/fillEvent';
+
+const schema = {
+  description: 'Creates a new fill event',
+  tags: ['Fill event'],
+  body: createFillEventBody,
+  response: {
+    201: fillEventResponse,
+    400: { $ref: 'error' },
+    401: { $ref: 'error' },
+    403: { $ref: 'error' },
+    500: { $ref: 'error' },
+  },
+};
+
```

Comment:
```suggestion

schema.body.properties.gasMixture.example = 'EAN21';

```

Kiintiöparannusehdotus :D